third_party/Makefile

```diff
@@ -30,3 +30,10 @@ ifneq (,$(wildcard cuda-samples/Samples/bandwidthTest/Makefile))
 	cd ./cuda-samples/Samples/bandwidthTest && make clean && make TARGET_ARCH=x86_64 SMS="70 75 80 86"
 	cp -v ./cuda-samples/Samples/bandwidthTest/bandwidthTest $(SB_MICRO_PATH)/bin/
 endif
+# Build nccl-tests.
+# The version we use is just the master branch, due to it didn't update for long time.
```

Comment:
When you add the submodule, the version is fixed as 44df0bf. You can say it is from one commit from master, but not the master branch.

Reply:
resolved in another pr